manifest.json

```diff
@@ -120,27 +138,27 @@
     {
       "name": "prospect_enabled",
       "title": "Enable Prospector",
-      "description": "Search for people in companies you Revealed or Discovered",
       "type": "boolean",
       "default": false
     },
     {
       "name": "handle_accounts",
-      "title": "Handle Accounts",
-      "description": "Save account objects instead of user objects in the clearbit_company group",
+      "title": "Use Accounts Feature (Beta)",
```

Comment:
@unity and @sbellity, I don't think that we should make this visible by default since we have not publicly announced the `account` feature. Thoughts?